### PR TITLE
Fix Travis CI builds and skip Fedora 31 i386 build/test cycles

### PR DESCRIPTION
**Pull request by prologic**
##### Summary

ssia

##### Component Name

- area/ci

##### Test Plan

- Travis CI

##### Additional Information

### CODE REVIEW

File: .travis/package_management/create_lxc_for_build.sh [hunk 1]
```diff
@@ -17,85 +17,90 @@ source .travis/package_management/functions.sh || (echo "Failed to load packagin
...
-	echo "No build release information defined. Make sure PACKAGE_TYPE is set on the environment before running this script"
-	exit 1
+  echo "No build release information defined. Make sure PACKAGE_TYPE is set on the environment before running this script"
+  exit 1
 fi
 
 # Detect architecture and load extra variables needed
 detect_arch_from_commit
 
+if [ x"${BUILDER_NAME}/${BUILD_STRING}/${BUILD_ARCH}" = "builder/fedora/31/i386" ]; then
```
Comment (Ferroin) [inline .travis/package_management/create_lxc_for_build.sh:54]:
```suggestion
if [ x"${BUILDER_NAME}/${BUILD_STRING}/${BUILD_ARCH}" = "xbuilder/fedora/31/i386" ]; then
```

Comment (prologic) [reply .travis/package_management/create_lxc_for_build.sh:54]: Gah you're right hold on 👍

Comment (prologic) [reply .travis/package_management/create_lxc_for_build.sh:54]: Fixed!

**Comment by prologic** [comment]
Merging to get the fix in today's nightly run (_in about ~1h15m_). I'll fix the Shellcheck warnings later.

**Pull request merged**